contracts/DAIInterestRateModelV3.sol

```diff
@@ -42,12 +42,16 @@ contract DAIInterestRateModelV3 is JumpRateModelV2 {
     }
 
     /**
-     * @notice Update the gapPerBlock parameter of the interest rate model (only callable by owner, i.e. Timelock)
-     * @param gapPerYear The additional margin per year separating the base borrow rate from the roof.
+     * @notice External function to update the parameters of the interest rate model
+     * @param baseRatePerYear The approximate target base APR, as a mantissa (scaled by 1e18). For DAI, this is calculated from DSR and SF. Input not used.
+     * @param gapPerYear The Additional margin per year separating the base borrow rate from the roof. (scaled by 1e18)
+     * @param jumpMultiplierPerYear The jumpMultiplierPerYear after hitting a specified utilization point
+     * @param kink_ The utilization point at which the jump multiplier is applied
      */
-    function updateGap(uint gapPerYear) public {
+    function updateJumpRateModel(uint baseRatePerYear, uint gapPerYear, uint jumpMultiplierPerYear, uint kink_) external {
         require(msg.sender == owner, "only the owner may call this function.");
-        gapPerBlock = gapPerYear / blocksPerYear;
+        gapPerBlock = gapPerYear.div(blocksPerYear);
```

Comment:
This div is actually not needed, as SafeMath only checks that the divisor is zero and we know it's not in this case!

Reply:
Yes you are right. My bad